mappings/net/minecraft/block/Material.mapping

```diff
@@ -11,8 +11,8 @@ CLASS net/minecraft/class_3614 net/minecraft/block/Material
 	FIELD field_15919 PORTAL Lnet/minecraft/class_3614;
 		COMMENT Material for the various portal blocks.
 	FIELD field_15920 WATER Lnet/minecraft/class_3614;
-	FIELD field_15921 UNUSED_PLANT Lnet/minecraft/class_3614;
-		COMMENT Not in use, but has foliage color.
+	FIELD field_15921 MOSS_BLOCK Lnet/minecraft/class_3614;
+		COMMENT Material for the moss block.
```

Comment:
```suggestion
		COMMENT Material for full sized moss blocks.
```
To match the pattern set by `SNOW_BLOCK`

Reply:
ah true